app/frontend/wallet/cardano-wallet.js

```diff
@@ -62,13 +62,13 @@ const CardanoWallet = async (options) => {
       debugLog(e)
       throw NamedError('TransactionRejectedByTrezor')
     })
+
     const response = await blockchainExplorer
       .submitTxRaw(signedTx.txHash, signedTx.txBody)
       .catch((e) => {
         debugLog(e)
         throw NamedError('TransactionRejectedByNetwork')
       })
-    updateUtxosFromTxAux(txAux)
```

Comment:
how is this replaced?

Reply:
thanks for pointing out, I forgot to reverse this change after debugging the tests